packages/bitcore-wallet-service/src/lib/chain/xrp/index.ts

```diff
@@ -136,7 +136,8 @@ export class XrpChain implements IChain {
     const recipients = outputs.map(output => {
       return {
         amount: output.amount,
-        address: output.toAddress
+        address: output.toAddress,
+        recipientTag: output.tag
```

Comment:
`tag` is already a param in some places for the destTag (e.g. CWC). Maybe we should keep that var name to simplify passing around instead of having to map between recipientTag <=> tag